docs/team/jovenpua.md

```diff
@@ -1 +1,65 @@
-Hi! My name is Joven, Y2 CS student from Malaysia. Not very sure what I want to do for now, so I’m going the AI path for now. I play a lot of games when I am free (Thanks uni!)
+---
+layout: page
+title: Joven Pua's Project Portfolio Page
+---
+
+## Project: Taskify
+
+Taskify is a desktop task management application for university students. Using Taskify, students can
+manage all their tasks (academics/personal/CCAs) effectively and seamlessly. The user interacts with it using a CLI,
+and it has a GUI created with JavaFX. It is written in Java, and has about 12 kLOC.
+
+Given below are my contributions to the project.
+
+* ### Major contributions:\
+    * #### Implemented multiple deletion of tasks
+        * What it does: Allows users to delete more than one task with one command, by providing multiple individual 
+          indices, giving an index range, or by providing a status of which all tasks that have it will be deleted.\
+          
+        * Justification: This feature makes deletion of tasks much faster for the user. Users would want to delete all 
+          completed tasks to declutter Taskify, or delete select tasks in Taskify without having to delete them one by
+          one.\
+          
+        * Highlights: The implementation involved rigorous testing because it was easy to mess up the rather complex 
+          regular expressions that accepts different types of input for the delete command. It was a little challenging
+          to reorganise the new code into their different classes to try to obey SLAP and SRP.\
+          
+        * Relevant PRs: [\#65](https://github.com/AY2021S2-CS2103T-W14-4/tp/pull/65), [\#158](https://github.com/AY2021S2-CS2103T-W14-4/tp/pull/158),
+          [\#160](https://github.com/AY2021S2-CS2103T-W14-4/tp/pull/160), [\#155](https://github.com/AY2021S2-CS2103T-W14-4/tp/pull/155),
+          [\#165](https://github.com/AY2021S2-CS2103T-W14-4/tp/pull/165), [\#172](https://github.com/AY2021S2-CS2103T-W14-4/tp/pull/172)  
+          
+    * #### Implemented the Date class
+        * What it does: An essential field in a Task, used for deadlines.\
```

Comment:
The `\` is appearing as part of the text. I think you should remove it.

Reply:
done